### PR TITLE
Bugfix/3.0.0

### PR DESCRIPTION
**Pull request by PeterMis**

### CODE REVIEW

File: lib/Controller/Display.php [hunk 1]
```diff
@@ -2275,6 +2275,10 @@ public function moveCms(Request $request, Response $response, $id)
                 throw new InvalidArgumentException(__('Provided CMS URL is invalid'), 'newCmsUrl');
             }
 
+            if (!v::stringType()->length(1, 1000)->validate($newCmsAddress)) {
```
Comment (dasgarner) [inline lib/Controller/Display.php:2278]: Doesn't a length from 1 to 1000 mean that we must enter a new CMS URL?

**Review by dasgarner — COMMENTED** [review]
Just a quick check on something

File: lib/Controller/Display.php (hunk 1 above)
Comment (PeterMis) [reply lib/Controller/Display.php:2278]: Well, yes, couple of lines above that we check if it's empty anyway ie it cannot be empty (or invalid or now over 1000 characters).

Is that not correct?

Comment (dasgarner) [reply lib/Controller/Display.php:2278]: Understood, makes sense if its always required! :+1:

**Review by dasgarner — APPROVED** [review]

**Pull request merged**